packages/server/lib/plugins/child/require_async_child.js

```diff
@@ -1,5 +1,26 @@
 process.title = 'Cypress: Config Manager'
 
+const { telemetry, OTLPTraceExporterIpc } = require('@packages/telemetry')
+
+const { file, projectRoot, telemetryCtx } = require('minimist')(process.argv.slice(2))
+
+let context
+let version
+
+if (telemetryCtx) {
+  const ctx = JSON.parse(
```

Comment:
should the `encode`/`decode` ctx methods live inside the telemetry package as utilities? Might be easier to navigate in the future as to what is doing the encoding/decoding

Reply:
moved